examples/vault/get.go

```diff
@@ -0,0 +1,62 @@
+// vault rotate is an example of how to use the rotate method
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/pangea"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/service/vault"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/service/audit"
+)
+
+func main() {
+	token := os.Getenv("PANGEA_VAULT_TOKEN")
+	if token == "" {
+		log.Fatal("Unauthorized: No token present")
+	}
+
+	auditTokenID := os.Getenv("PANGEA_AUDIT_TOKEN_ID")
+	if auditTokenID == "" {
+		log.Fatal("Unauthorized: No audit token id present")
+	}
+    vaultConfig := pangea.Config{
+		Token:  token,
+		Domain: os.Getenv("PANGEA_DOMAIN"),
+	}
+	vaultObj := vault.New(&vaultConfig)
```

Comment:
We usually call it `client` instead of object

Reply:
it was called *cli before which normally stands for command line which i found confusing, i dont mind calling it "client" tho